synapse/tools/easycert.py

```diff
@@ -67,7 +68,7 @@ def main(argv, outp=None):
             return 0
 
         if opts.server:
-            cdir.genHostCert(opts.name, signas=opts.signas, outp=outp)
+            cdir.genHostCert(opts.name, signas=opts.signas, outp=outp, sans=opts.server_sans)
```

Comment:
its correct - something must've rendered runny with chrome and it looked like there were extraneous spaces.